qa/suites/fs/one-role/tasks/mgr-osd-full.yaml

```diff
@@ -0,0 +1,21 @@
+overrides:
+  ceph:
+    conf:
+      global:
+        osd_pool_default_size: 1
+        osd_pool_default_min_size: 1
+      client:
+        debug ms: 1
+        debug client: 20
+        client check pool perm: false
```

Comment:
This should be on the `mgr:`, right?

If that's a requirement for the mgr now, I suggest adding a config to libcephfs to modify the behavior so the mgr can override. The mgr/volumes plugin would modify the option for every libcephfs connection it creates.

Reply:
Done